src/index.ts

```diff
@@ -359,22 +362,25 @@ const generateParachainGenesisFile = (
  * @param config
  * @param output
  * @param yes
+ * @param relaychainOnly
  */
-const generateDockerfiles = (config: Config, output: string, yes: boolean) => {
+const generateDockerfiles = (config: Config, output: string, yes: boolean, relaychainOnly: boolean) => {
```

Comment:
I'm not sure yet if we really need this flag, because if `parachains.length === 0`, then it simply implies this